### PR TITLE
Updated transform_all.bat

### PR DESCRIPTION
**Pull request by zeusongit**
Edited transform_all.bat to work with vs2019, utilizing vswhere.exe utility.

Please Note:
1. Before submitting the PR, please review [How to Contribute to Dynamo](https://github.com/DynamoDS/Dynamo/blob/master/CONTRIBUTING.md)
2. Dynamo Team will meet 1x a month to review PRs found on Github (Issues will be handled separately)
3. PRs will be reviewed from oldest to newest
4. If a reviewed PR requires changes by the owner, the owner of the PR has 30 days to respond. If the PR has seen no activity by the next session, it will be either closed by the team or depending on its utility will be taken over by someone on the team
5. PRs should use either Dynamo's default PR template or [one of these other template options](https://github.com/DynamoDS/Dynamo/wiki/Choosing-a-Pull-Request-Template) in order to be considered for review.
6. PRs that do not have one of the Dynamo PR templates completely filled out with all declarations satisfied will not be reviewed by the Dynamo team.
7. PRs made to the `DynamoRevit` repo will need to be cherry-picked into all the DynamoRevit Release branches that Dynamo supports. Contributors will be responsible for cherry-picking their reviewed commits to the other branches after a `LGTM` label is added to the PR.

### Purpose

https://jira.autodesk.com/browse/DYN-2152

AssemblySharedInfo.tt file was not being updated with the latest build no., as the paths for vs were not defined.
Now, by using vswhere tool, vs path is provided irrespective of the version used to build the project.

### Declarations

Check these if you believe they are true

- [ ] The code base is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions), and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.

### Reviewers

@DynamoDS/dynamo

### CODE REVIEW

**Comment by QilongTang** [comment]
LGTM

File: src/transform_all.bat [hunk 1]
```diff
@@ -17,24 +17,30 @@ dir *.tt /b > t4list.txt
...
+:: clear text transform path to undefine it
+set TEXTTRANSFORMPATH=
 
-IF NOT EXIST "%TextTransform%" set TextTransform=%COMMONPROGRAMFILES(x86)%\microsoft shared\TextTemplating\14.0\TextTransform.exe
-
-IF NOT EXIST "%TextTransform%" set TextTransform=%ProgramFiles(x86)%\Microsoft Visual Studio\2017\Professional\Common7\IDE\TextTransform.exe
-
-IF NOT EXIST "%TextTransform%" set TextTransform=%ProgramFiles(x86)%\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\TextTransform.exe
+:: use latest vswhere utility to locate in IDE, where it resides now. 
+IF EXIST "%ProgramFiles(x86)%\Microsoft Visual Studio\Installer\vswhere.exe" (
```
Comment (mjkkirschner) [inline src/transform_all.bat]: do you happen to know if this always exists? Even on machines with only the vs build tools installed? 
https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2019

**Comment by mjkkirschner** [comment]
@zeusongit @QilongTang @aparajit-pratap  do you think the build should fail if this transform fails? Right now it does not.

File: src/transform_all.bat (hunk 1 above)
Comment (zeusongit) [reply src/transform_all.bat]: Yes, vswhere.exe should always be present, with just vs build tools or with VS 2015.2 Preview 2 and ahead.
This is the issue regarding the vswhere, where they decided to put it in a unified location:
*https://github.com/microsoft/vswhere/issues/41*

Comment (mjkkirschner) [reply src/transform_all.bat]: great, thanks.

**Comment by QilongTang** [comment]
I dont think the build should fail for everyday user.. 3rd party dev should not worry about this I think. @mjkkirschner @aparajit-pratap What do you think?

**Comment by QilongTang** [comment]
I am going to merge this, at the same time uninstalling VS2017 locally on my machine to test this

**Pull request merged**